com.moddb.TotalChaosRetro.appdata.xml

```diff
@@ -36,6 +36,6 @@
     <content_attribute id="drugs-tobacco">moderate</content_attribute>
   </content_rating>
   <releases>
-    <release date="2022-08-08" version="1.40"/>
+    <release date="2022-08-08" version="1.40"><url>https://www.moddb.com/mods/total-chaos/downloads/total-chaos-directors-cut-retro-edition-140</url></release>
```

Comment:
Just a small nitpick. I would probably improve a formatting of this line a little bit.

Reply:
In [other projects](https://github.com/xbmc/xbmc/blob/master/tools/Linux/kodi.metainfo.xml.in) I see no formatting, maybe to have just one line for each version. But it's ok to format, since I don't think we'll have other versions.